src/components/IssueController.jsx

```diff
@@ -61,7 +61,7 @@ export default class IssueController extends FalcorController {
           //console.log("Category: " + JSON.stringify(category));
           return(
             <div key={category} className="issue__category">
-              <h2 className="section-header">{category}</h2>
+              <h2 className="section-header">{category.replace('-', ' ')}</h2>
```

Comment:
Okay that's fine but if we start doing this more, we should have a function
"categorySlugToName". I'm just worried about developers doing it
inconsistently or not knowing about this.

On Sun, Aug 7, 2016, 4:09 AM Zane Mountcastle <notifications@github.com>
wrote:

> In src/components/IssueController.jsx
> <https://github.com/thegazelle-ad/gazelle-front-end/pull/41#discussion_r73794635>
> :
>
> > @@ -61,7 +61,7 @@ export default class IssueController extends FalcorController {
> >            //console.log("Category: " + JSON.stringify(category));
> >            return(
> >              <div key={category} className="issue__category">
> > -              <h2 className="section-header">{category}</h2>
> > +              <h2 className="section-header">{category.replace('-', ' ')}</h2>
>
> It's only replacing dashes with spaces. You think it's work the extra work
> with Falcor and fetching the data essentially twice?
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/thegazelle-ad/gazelle-front-end/pull/41/files/a85a19c83b836058745a11879e25ef14a8a7a67e#r73794635>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADAFzfpMwwiMPHGJLS4MchGarwh1CgKGks5qdT5YgaJpZM4JdXZj>
> .
>